utils/barreleye.py

```diff
@@ -1,6 +1,7 @@
 # Like eyeing fish in a barrel!
 
 import requests
+from utils.custom_drinks import add_custom_drinks, add_custom_recipe, CUSTOM_DRINK_LIST
```

Comment:
its' a nitpick, but maybe we can shorten this up a bit

`import utils.custom_drinks as cd`

then refer to the imported stuff as `cd.add_custom_crinks` etc.